documentation/manual/src/main/asciidoc/modules/query.asciidoc

```diff
@@ -421,3 +422,73 @@ go check the https://docs.jboss.org/hibernate/search/{hibernate-search-major-min
 === Using the Criteria API
 
 At this time, we have not implemented support for the Criteria APIs (neither Hibernate native nor JPA).
+
+=== Using stored procedures
+
+Often you want the raw power of the underlying NoSQL query engine.
+Even if that costs you portability.
+
+Hibernate OGM addresses that problem by letting you express stored procedures (e.g. in Server-side JavaScript for MongoDB )
```

Comment:
* Let's drop the cap on `Server-side`, `server-side` it is.
 * `MongoDB )`, we should remove the space before the parenthesis